libraries/WiFi/src/WiFiMulti.cpp

```diff
@@ -261,12 +269,10 @@ bool WiFiMulti::testConnection(int32_t i){
     WiFiClient client;
     const int httpPort = 80;
```

Comment:
port needs to be dynamically determined from the test url, it is not guaranteed that all servers will expose port 80 (most are moving to https as a default)

Reply:
Hmm.  Can I make an https request directly from wifiClient?  I'm trying to keep it simple by not including more libraries than necessary.  I'm assuming anyone using this function would set the test page manually.  Still, if you feel this is important I'll modify it.

Reply:
true, but if a user sets a url with https it would likely fail here as well

Reply:
Yeah.  In my example I specify that it must be http://.  Need to think it through a bit more.